src/DonationForms/V2/ListTable/Columns/GoalColumn.php

```diff
@@ -49,9 +49,11 @@ public function getCellValue($model): string
             return __('No Goal Set', 'give');
         }
 
-        add_filter('give_get_form_earnings_stats', function ($earnings, $donationFormId) {
-            return (new DonationQuery())->form($donationFormId)->sumAmount();
-        }, 10, 2);
+        $usePlaceholder = apply_filters('givewp_admin_list_form_view_goal_progress_stats_use_placeholder', false);
```

Comment:
You could drop an action here and hook in outside of the column so the placeholder logic is not so coupled:

```
do_action("givewp_list_table_cell_value_{$this::getId()}_before", $model);
```

(or we could even add to the ListTable framework)

Reply:
Resolved in 149593253aa596f97009b26db3a96e69221de70c and 5992e441fd4870514cf99cb76116217ccdccf746